src/Components/App/App.js

```diff
@@ -1,24 +1,33 @@
+import React from 'react';
 import './App.css';
 import Game from '../Game/Game'
 import Header from '../Header/Header';
 import {Route, Routes} from 'react-router-dom'
-import Question from '../Question/Question';
 import Category from '../Category/Category'
 import Difficulty from '../Difficulty/Difficulty'
 import ErrorDisplay from '../ErrorDisplay/ErrorDisplay'
+import { useState } from 'react';
+
+export const gameContext = React.createContext()
```

Comment:
Can we import `createContext` at the top of our file instead of using dot notation?  Just for consistency and to make things a little cleaner.